example/material_tooltip_example/lib/material_tooltip_example.dart

```diff
@@ -16,16 +17,19 @@ import 'package:angular_gallery_section/annotation/gallery_section_config.dart';
 import 'package:angular_gallery_section/components/content/delayed_content.dart';
 import 'package:angular_components/theme/dark_theme.dart';
 
-@GallerySectionConfig(displayName: 'Material Tooltip', docs: const [
-  MaterialTooltipDirective,
-  MaterialPaperTooltipComponent,
-  MaterialTooltipTargetDirective,
-  ClickableTooltipTargetDirective,
-  MaterialInkTooltipComponent,
-  MaterialIconTooltipComponent
-], demos: const [
-  MaterialTooltipExampleComponent
-])
+@GallerySectionConfig(
+  displayName: 'Material Tooltip',
+  docs: const [
+    MaterialTooltipDirective,
+    MaterialPaperTooltipComponent,
+    MaterialTooltipTargetDirective,
+    ClickableTooltipTargetDirective,
+    MaterialInkTooltipComponent,
+    MaterialIconTooltipComponent
+  ],
+  demos: const [MaterialTooltipExampleComponent],
+  showGeneratedDocs: true,
+)
```

Comment:
In `@GallerySectionConfig()` of this file and a few others: you shouldn't need the `const` keywords anymore.

Reply:
We haven't started cleaning these up internally yet. We were told to wait until some edge cases were solved internally. I'd rather not do them piecewise, but cleanup a large swatch when we get the all clear.

Reply:
SGTM